javascript/chronometer.js

```diff
@@ -1,34 +1,64 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = null;
+
   }
 
   start(callback) {
-    // ... your code goes here
+    
+    if (typeof callback == "undefined"){
+      this.intervalId = setInterval(()=>{
+          this.currentTime += 1;
+      },1000)
+    }else{
+      callback;
+    }
   }
 
+
   getMinutes() {
-    // ... your code goes here
+    if (this.currentTime == 0){
+      return 0
+    }
+    if(this.currentTime > 59){
+      return (this.currentTime - (this.currentTime % 60))/60
+    }else{
+      return 0
+    }
   }
 
   getSeconds() {
-    // ... your code goes here
+    if (this.currentTime == 0){
+      return 0
+    }else{
+      return this.currentTime%60
+    }
   }
 
   computeTwoDigitNumber(value) {
-    // ... your code goes here
+    value = value.toFixed()
+    if (value.length === 1 ){
+      value = "0" + value
+      return value
+    }
+    return value
```

Comment:
Legal essa solução!
Outra opção é, antes de transformar o value em string, fazer a verificação: se for menor que 10, retornar adicionando o 0, se não, retornar o valor transformado em string.